packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -367,19 +397,7 @@ export class StripeWebhookHandler extends StripeHandler {
         return;
       }
       reportSentryError(err, request);
-      return;
     }
-    await this.capabilityService.stripeUpdate({ sub, uid });
-    if (this.paypalHelper) {
-      const customer = await this.stripeHelper.fetchCustomer(uid, [
-        'subscriptions',
-      ]);
-      if (!customer || customer.deleted) {
-        return;
-      }
-      return this.paypalHelper.conditionallyRemoveBillingAgreement(customer);
```

Comment:
I noticed the previous code returned this value, whereas now we're calling the method without returning. Looking at the method, it returns a boolean based on whether it removed the billing agreement or not. It looks like this is the only caller, and we didn't do anything with this return value, so it seems fine.